README.md

```diff
@@ -97,7 +102,37 @@ Each lion should have:
 Create a new lion instance with the name `simba`
 
 ```ruby
-# code here
+
+
+class Animal
+  attr_accessor :name
+
+  def initialize
+    @name = name
+  end
+
+  def greet
+    puts "Hi, my name is #{@name}!"
+  end
+end
+
+pumba = Animal.new("Pumba")
+
+class Lion < Animal
+  @@pack = []
+  @@king = false
+
+  def initialize
+    @@pack.push(self)
+  end
+
+  def king (@@king)
+    if @name = "Simba"
+      @@king = true
+    end
+end
+
```

Comment:
we talked about this too, but essentially put this in your initialize and start off with `@king = false`. make sure you're using an instance variable not class, and review the definitions of both